src/net.cpp

```diff
@@ -2496,6 +2496,7 @@ void CConnman::ThreadOpenMasternodeConnections()
             return pending;
         };
 
+        constexpr const auto &_func_ = __func__;
```

Comment:
this line is no longer need now that #4934 is merged https://github.com/dashpay/dash/commit/17b15efd61338d67653d2eca8802c5cd6fa7f526#diff-00021eed586a482abdb09d6cdada1d90115abe988a91421851960e26658bed02R2421
```suggestion
```